doc/source/reference/arrays.datetime.rst

```diff
@@ -309,6 +309,46 @@ us / μs    microsecond      +/- 2.9e5 years         [290301 BC, 294241 AD]
    as      attosecond       +/- 9.2 seconds         [  1969 AD,   1970 AD]
 ======== ================ ======================= ==========================
 
+
+Converting datetime and timedelta to Python Object
+==================================================
+
+NumPy follows a strict protocol when converting datetime64 and/or timedelta64 to Python Objects (e.g., tuple, list, datetime.datetime). 
+
+The protocol is described in the following table:
+
+================================ ================================= ==================================
+         Input Type                         for datetime64                    for timedelta64
+================================ ================================= ==================================
+              NaT                               None                               None
```

Comment:
```suggestion
            ``NaT``                           ``None``                           ``None``
```

similarly for all the other python types in the heading and the rows below as well as the text. You don't need to do it in the code examples because those are already interpreted as Python code.